data-manip.Rmd

```diff
@@ -1,10 +1,9 @@
 ---
 title: Data transformation
-output: oldbookdown::html_chapter
-bibliography: references.bib
+knit: "bookdown::preview_chapter"
 ---
 
-```{r data, include = FALSE}
+```{r data-manip, include = FALSE}
 chapter <- "data-manip"
```

Comment:
I think we should also stop using `chapter`, and remove the custom `fig.path` and `cache.path` in `common.R`.